spec/services/file_storage_service_spec.rb

```
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe FileStorageService do
+  describe ".duplicate_blob" do
+    it "creates a duplicate of an existing file" do
+      user = create(:user)
+      product = create(:digital_product, user: user)
+      
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)